ts/Core/Options.ts

```diff
@@ -3747,7 +3747,7 @@ H.defaultOptions = {
 
         /**
          * The name of a symbol to use for the border around the tooltip. Can
-         * be one of: `"callout"`, `"circle"`, or `"square"`. When
+         * be one of: `"callout"`, `"circle"`, `"square"` or `"rect"`. When
```

Comment:
Actually `square` doesn't make sense for the tooltip border anymore. It is a rectangle, not a square. Should we just remove this from the description? It will still work, but we don't have to encourage it by listing it here.

Reply:
But wont typescript users with `square` tooltips get complaints from the compiler then? Do we expect them to just replace it with `rect`?

Reply:
Yes good point. I suggest we remove it from the description, but leave it in the type.